### PR TITLE
Implement DLB CI

### PR DESCRIPTION
**Pull request by bart0sh**
This PR is going to contain everything needed to enable triggering DLB e2e testing on simulated SPR target

Fixes: #948

### CODE REVIEW

File: demo/dlb-libdlb-demo/test.sh [hunk 1]
```diff
@@ -3,8 +3,7 @@
 dlb_dev=$(ls /dev/dlb* | sed 's/\/dev\/dlb//' | head -1)
 echo '\n1. Directed Traffic test'
 echo '--------------------------'
-/usr/local/bin/dir_traffic -n 128 -d $dlb_dev
+/usr/local/bin/dir_traffic -n 8 -d $dlb_dev -w epoll
 echo '\n2. Load Balanced Traffic test'
 echo '-------------------------------'
-/usr/local/bin/ldb_traffic -n 128 -d $dlb_dev
-
+/usr/local/bin/ldb_traffic -n 8 -d $dlb_dev -w epoll
```
Comment (hj-johannes-lee) [inline demo/dlb-libdlb-demo/test.sh:9]: nitpick

**Review by hj-johannes-lee — CHANGES_REQUESTED** [review]

File: demo/dlb-libdlb-demo/test.sh (hunk 1 above)
Comment (bart0sh) [reply demo/dlb-libdlb-demo/test.sh:9]: ???

**Review by ozhuraki — APPROVED** [review]
LGTM

**Review by mythi — APPROVED** [review]
looks OK to me. 

@hj-johannes-lee do you want any changes still?

**Review by hj-johannes-lee — APPROVED** [review]
LGTM! :D

File: demo/dlb-libdlb-demo/test.sh (hunk 1 above)
Comment (hj-johannes-lee) [reply demo/dlb-libdlb-demo/test.sh]: @bart0sh Sorry, if you didn't understand.! But it seems that you changed! :)

**Pull request merged**